doc/syscalls/70006_lsm303dlhc.md

```diff
@@ -0,0 +1,129 @@
+---
+driver number: 0x70006
+---
+
+# LSM303DLHC
+
+## Overview
+
+Three axis accelerometer, magnetometer and temperature sensor.
+
+[Manual](https://www.st.com/resource/en/datasheet/lsm303dlhc.pdf)
+
+## Command
+
+  * ### Command number: `0`
+
+    **Description**: Returns SUCCESS
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: 0
+
+  * ### Command number: `1`
+
+    **Description**: Performs a test read/write to verify that the hardware sensor is present.
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if presence test has started successfully, `EBUSY` otherwise.
+
+  * ### Command number: `2`
+
+    **Description**: Set Accelerometer Power Mode.
+
+    **Argument 1**: Accelerometer Data rate defined in manual table 20, page 25
+
+    **Argument 2**: Low power mode (1 on, 0 off) 
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+  * ### Command number: `3`
+
+    **Description**: Set Accelerometer Scale and Resolution.
+
+    **Argument 1**: Accelerometer scale defined in manual table 27, page 27
+
+    **Argument 2**: High resolution (1 on, 0 off)
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+  * ### Command number: `4`
+
+    **Description**: Set Magnetometer Temperature Enable and Data Rate.
+
+    **Argument 1**: Temperature enable (1 on, 0 off)
+
+    **Argument 2**: Magnetometer Data rate defined in manual table 72, page 37
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+    
+  * ### Command number: `5`
+
+    **Description**: Set magnetometer range.
+
+    **Argument 1**: Magnetometer range defined in manual table 75, page 38
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+  * ### Command number: `6`
+
+    **Description**: Reads Acceleration X, Y and Z
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+  * ### Command number: `7`
+
+    **Description**: Reads the temperature
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+  * ### Command number: `8`
+
+    **Description**: Reads Magnetometer X, Y and Z
+
+    **Argument 1**: unused
+
+    **Argument 2**: unused
+
+    **Returns**: `SUCCESS` if there is no other command in progress, `EBUSY` otherwise.
+
+## Subscribe
+
+All the commands return a callback when done.
+
+  * ### Subscribe number `0`
+
+    **Description**: Called when commands are done
```

Comment:
<img width="528" alt="image" src="https://user-images.githubusercontent.com/339422/79894655-c71b1380-83d3-11ea-8067-24f67959afda.png">
nit: something is wonky with the whitespace here

Reply:
I think this is from my editor, is there any guideline for the md?

Reply:
By driver do you mean the i2c file from each chip? Wouldn't this require a change in all the chips, vs the current solution of implementing it in the virtual_i2c device?

Reply:
Any feedback here so we can we can merge all the functionalities for the discovery?